services/ux-backend/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handler"
+)
+
+func main() {
+
```

Comment:
You should load and decode the onboarding private key here and abort the server if you cannot.

Reply:
Okay